bindings/py/setup.py

```diff
@@ -200,4 +200,9 @@ def generateExtensions():
       "Intended Audience :: Science/Research",
       "Topic :: Scientific/Engineering :: Artificial Intelligence"
     ],
+    entry_points = {
+      "console_scripts": [
+        "nupic-check-bindings = nupic.bindings.check:printImportBindingsMessage",
```

Comment:
Nit: might be better to flip the command line name to "nupic-bindings-check" so that if we add others in the future they can all start with "nupic-bindings-" but up to you.